spec/tasks/mongoid_slug_rake_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+require 'rake'
+
+describe 'mongoid_slug:set' do
+  before :all do
+    load File.expand_path('../../../lib/tasks/mongoid_slug.rake', __FILE__)
+    Rake::Task.define_task(:environment)
+  end
+
+  context 'when models parameter is passed' do
+    let!(:uninitalized_slug1) { UninitalizedSlugFirst.create(name: 'uninitalized_slug1') }
+    let!(:uninitalized_slug2) { UninitalizedSlugSecond.create(name: 'uninitalized_slug2') }
+
+    it 'Goes though all documents of passed models and sets slug if not already set' do
+      class UninitalizedSlugFirst
```

Comment:
You have these redefined here, they are already in `spec/models`. 

If you define them inline (which could be fine, too), use:

```ruby
let(:uninitialized_slug_klass) { Class.new do
    include Mongoid::Slug
    slug :name
end
}
```

Might need a `store_in` or something like that since the class doesn't have a name.